config/Upgrade/services.yaml

```diff
@@ -1,3 +1,8 @@
+parameters:
+    report_send_endpoint_url: https://data.prod.spryker-dwh.net/upgrader
```

Comment:
Do we have a dev or test environment for that? Or only prod?

Reply:
Need to start using symfony env for that. As for me it should be in separate task. By default reports are not sent